src/validation.cpp

```diff
@@ -147,6 +146,13 @@ namespace {
     std::set<int> setDirtyFileInfo;
 } // anon namespace
 
+CBlockIndex* LookupBlockIndex(const uint256& hash)
```

Comment:
It's unclear reading commits messages what's the long term destination of `g_blockman` is. Do other parts of the code calling `LookupBlockIndex` will always rely on a global `BlockManager` or should they access to it through their instance of `CChainstate`  after future refactoring ?

Reply:
Eventually, a single `BlockManager` object will be managed and shared to all created `CChainState` objects by something called `ChainstateManager`, to be introduced later (https://github.com/bitcoin/bitcoin/pull/15606/commits/df81dc462e72047ed6857d927946cb18e0e9492b). The single global block index will be accessed through the chainstate manager.